benchmark/src/client.rs

```diff
@@ -254,6 +256,38 @@ impl<B: Backoff + Send + 'static> RequestExecutor<B> {
         };
         spawn!(client, keep_running, "streaming ping pong", f);
     }
+
+    fn execute_stream_from_client(mut self) {
```

Comment:
Can you reference the original implement?

Reply:
```c++
  bool ThreadFuncImpl(HistogramEntry* entry, size_t thread_idx) override {
    // Figure out how to make histogram sensible if this is rate-paced
    if (!WaitToIssue(thread_idx)) {
      return true;
    }
    GPR_TIMER_SCOPE("SynchronousStreamingFromClientClient::ThreadFunc", 0);
    if (stream_[thread_idx]->Write(request_)) {
      double now = UsageTimer::Now();
      entry->set_value((now - last_issue_[thread_idx]) * 1e9);
      last_issue_[thread_idx] = now;
      return true;
    }
    stream_[thread_idx]->WritesDone();
    FinishStream(entry, thread_idx);
    auto* stub = channels_[thread_idx % channels_.size()].get_stub();
    std::lock_guard<std::mutex> l(stream_mu_[thread_idx]);
    if (!shutdown_[thread_idx].val) {
      stream_[thread_idx] = stub->StreamingFromClient(&context_[thread_idx],
                                                      &responses_[thread_idx]);
    } else {
      stream_[thread_idx].reset();
      return false;
    }
    return true;
  }
```
grpc/test/cpp/qps/client_sync.cc line:302~327

Reply:
In fact, when I wrote bench at the time, I did not fully refer to the implementation of grpc-cpp

Reply:
> ...I did not fully refer to the implementation of grpc-cpp

You should. It's expected to work across implement.